Mattvol4 (#118)
rapid7/insightconnect-plugins: dumbno/help.md

# Description

> [Dumbno](https://github.com/ncsa/dumbno) is free and open source flow shunting software for Arista switches using EOS API. The InsightConnect plugin can add ACLs to the switches 

Reviewer: ```suggestion
[Dumbno](https://github.com/ncsa/dumbno) is a free and open source flow shunting software for Arista switches using the EOS API. Using this plugin will allow users to add ACLs to switches
```

> [Dumbno](https://github.com/ncsa/dumbno) is free and open source flow shunting software for Arista switches using EOS API. The InsightConnect plugin can add ACLs to the switches 

Reviewer 2: ```suggestion
[Dumbno](https://github.com/ncsa/dumbno) is a free and open source flow shunting software for Arista switches using EOS API. The InsightConnect plugin can add ACLs to the switches 
```
